SConstruct

```diff
@@ -168,11 +156,7 @@ if profile:
 opts = Variables(customs, ARGUMENTS)
 
 # Target build options
-if env.scons_version >= (4, 3):
-    opts.Add(["platform", "p"], "Target platform (%s)" % "|".join(platform_list), "")
-else:
-    opts.Add("platform", "Target platform (%s)" % "|".join(platform_list), "")
-    opts.Add("p", "Alias for 'platform'", "")
+opts.Add((["platform", "p"], "Target platform (%s)" % "|".join(platform_list), ""))
```

Comment:
That breaks support for SCons < 4.3.0.

Reply:
I'm 99% sure it doesn't if wrapping in a tuple, but this probably goes outside the scope of a bugfix. I'll make a simpler version

Reply:
If it works with a tuple, then that's much nicer than the current hack indeed. My impression was that only SCons 4.3.0+ supported aliases.

Reply:
Tested with SCons 4.0.0, I confirm it works:
```
$ scons --help
...
platform: Target platform (linuxbsd|windows)
    default: 
    actual: linuxbsd
    aliases: ['p']
```

So I withdraw my objection here :)